smithy.yaml

```diff
@@ -1,12 +1,15 @@
 project: dart
 language: dart
 
-# dart 1.19.1
-runner_image: drydock-prod.workiva.org/workiva/smithy-runner-generator:92530
+# This image is created using https://github.com/Workiva/smithy-runner-generator
+runner_image: drydock-prod.workiva.net/workiva/smithy-runner-generator:179735 # Dart 1.24.2
```

Comment:
If we're bumping the image, should we also bump the min SDK version to `^1.24.0`?